modules/imgproc/src/drawing.cpp

```diff
@@ -1457,16 +1459,16 @@ FillEdgeCollection( Mat& img, std::vector<PolyEdge>& edges, const void* color )
                     te->next = last;
                     prelast = te;
                     sort_flag = 1;
+                    last_exchange = prelast;
```

Comment:
Perhaps we don't need `sort_flag` anymore.

`sort_flag == 1` iff `last_exchange != NULL`

---

```
if (last_exchange == NULL)
    break;
```

before "while"

Reply:
Yes, that's right

Reply:
@zcx-xxx Could you please to update PR?

Reply:
Sorry, I haven't read it recently. The PR has been updated